src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -189,18 +184,13 @@ function useCheckEncryptionState(state: State, setState: (state: State) => void)
         if (state === "loading") checkEncryptionState();
     }, [checkEncryptionState, state]);
 
-    useTypedEventEmitter(matrixClient, ClientEvent.AccountData, (event: MatrixEvent): void => {
-        const type = event.getType();
-        // Recheck the status if this account data has been updated as this implies the status
-        // of megolm key backup has changed on the user's account (there's no event emitted for megolm
-        // key backup enabled state changing, so we use this instead).
+    useTypedEventEmitter(matrixClient, CryptoEvent.KeyBackupStatus, (): void => {
+        // Recheck the status if the key backup status has changed so we can keep the page up to date.
         // Note that this could potentially update the UI while the user is trying to do something, although
         // if their account data is changing then it implies that they're changing encryption related things
         // on another device. This code is written with the assumption that it's better for the UI to refresh
         // and be up to date with whatever changes they've made.
```

Comment:
this still talks about account data, which I think is incorrect

Reply:
Oops, good catch.